dspace-spring-rest/src/main/java/org/dspace/app/rest/model/RelationshipRest.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.dspace.app.rest.RestResourceController;
+
+/**
+ * This class acts as the REST representation of the Relationship model class.
+ * This class acts as a data holder for the RelationshipResource
+ * Refer to {@link org.dspace.content.Relationship} for explanation about the properties
+ */
+public class RelationshipRest extends BaseObjectRest<Integer> {
+    public static final String NAME = "relationship";
+    public static final String CATEGORY = "core";
+
+    private UUID leftId;
+    private int relationshipTypeId;
```

Comment:
The int relationshipTypeId is currently still present as part of a transition to the HAL link. This will be removed once all usages of the int ID are removed as well